### ISSUE TITLE
"Show periods" links do not have unique id

### ISSUE THREAD

**Issue by kburk1997** [YACS-RCOS/yacs-admin#191]
Each "Show periods" link has the same ID:
`<a id="showPeriods">`.

Assistive technologies will overlook elements with non-unique IDs, so this should be fixed ASAP.

**Comment by Alfriadox** [comment]
Mine
